app/pages/project/about/index.cjsx

```diff
@@ -0,0 +1,63 @@
+React = require 'react'
+TitleMixin = require '../../../lib/title-mixin'
+{Markdown} = require 'markdownz'
+PromiseRenderer = require '../../../components/promise-renderer'
+Translate = require 'react-translate-component'
+counterpart = require 'counterpart'
+{Link} = require 'react-router'
+
+counterpart.registerTranslations 'en',
+    nav:
+        research: 'Research'
+        results: 'Results'
+        faq: 'FAQ'
+        education: 'Education'
+        team: 'The Team'
+
+module.exports = React.createClass
+  displayName: 'About'
+
+  mixins: [TitleMixin]
+
+  title: 'About'
+
+  getDefaultProps: ->
+    project: null
+
+  getPageTitles: (page) ->
+    page.filter((page) -> page.content isnt '' and page.content?)
+      .reduce(((accum, page) -> accum[page.url_key] = page.title; accum), {})
+
+  render: ->
+    [ownerName, name] = @props.project.slug.split('/')
+    projectPath = "/projects/#{ownerName}/#{name}"
```

Comment:
You don't need to manually split the slug, just pass it in as `"/projects/#{@props.project.slug}"`. The slug is already formatted correctly to use in URL paths.